integration/cmd/osrm-files-extractor/main.go

```diff
@@ -34,6 +36,12 @@ const (
 
 	dotCNBGSuffix        = ".cnbg"
 	dotOSRMDotCNBGSuffix = dotOSRMSuffix + dotCNBGSuffix
```

Comment:
Just an idea: May be those code is better stay with each package.
- predefinition
- registration